src/containers/TemplateBuilder/template/Actions/ActionConfig.tsx

```diff
@@ -133,6 +135,16 @@ const ActionConfig: React.FC<ActionConfigProps> = ({ templateAction, onClose })
           <div className="spacer-10" />
           <div className="config-container-outline">
             <div className="flex-column-start-center">
+              <TextIO
```

Comment:
Would it have been ok to just:

`text={state.code || ''}`
`setState({ ..state: code: test || null}))` here ? 
Sorry I have no idea what the side effects are of chaging `TextIOprops` for text to string | null, it seems easier just to map null to '' for text and '' to null when saving back to state

Reply:
also saw this in console (I think it's related '`value` prop on `input` should not be null'

Reply:
> Would it have been ok to just:
> 
> `text={state.code || ''}` `setState({ ..state: code: test || null}))` here ?

I think I tried something like that at first, but then it caused [another problem](https://github.com/openmsupply/conforma-web-app/pull/1224#discussion_r924119245) in that it wouldn't save empty values at all if changing them from existing text content.

I'll have another look at though, there should be a better way.

Reply:
Yeah, that worked better. Thanks for the suggestion: 9c6e82a1

And I was able to have them fall back to either `null` or `""` depending on whether or not the database field is nullable.

Reply:
thanks